index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
Always ensure to validate your HTML code using [https://validator.w3.org/nu/#textarea](https://validator.w3.org/nu/#textarea) and address any warnings or errors that are present.